packages/sysdig/data_stream/event/manifest.yml

```
@@ -0,0 +1,93 @@
+title: Security Event
+type: logs
+streams:
+  - input: cel
+    title: Security Event
+    description: Collecting security events via API.
+    template_path: cel.yml.hbs
+    enabled: false
+    vars:
+      - name: initial_interval
+        type: text
+        title: Initial Interval
+        multi: false
+        required: true
+        show_user: true
+        default: 24h
+        description: How far back to pull the security events from Sysdig API. Supported units for this parameter are h/m/s.
+      - name: interval
+        type: text
+        title: Interval
+        description: Duration between requests to the Sysdig API. Supported units for this parameter are h/m/s.
+        default: 5m
+        multi: false
+        required: true
+        show_user: true
+      - name: batch_size
+        type: integer
+        title: Batch Size
+        multi: false
+        required: true
+        show_user: false
+        description: Batch size for the response of the Sysdig API. The maximum supported page size value is 200.
+        default: 200
+      - name: max_executions
+        type: integer
+        title: Max Executions
+        description: Max Executions is the maximum number of times a CEL program can request to be re-run with a want_more field.
```

```suggestion
      - name: max_executions
        type: integer
        title: Maximum Pages Per Interval
        description: Maximum Pages Per Interval is the maximum number of pages that can be collected at each interval.
```
Executions is an implementation detail that the user should not care about. They do care about what the implications are though, which is the number of pages that can be collected. Please also update the reference to this in the README text.